src/Microsoft.DotNet.Interactive.Http.Parsing/Parsing/HttpRequestParser.cs

```diff
@@ -462,6 +474,11 @@ private void ConsumeCurrentTokenInto(HttpSyntaxNode node)
                     }
                 }
 
+                if (!((CurrentToken?.Kind is HttpTokenKind.Word or HttpTokenKind.Punctuation) || (IsAtStartOfEmbeddedExpression())))
```

Comment:
Was this another bug in the url parsing? Could you please add a separate test for this?